### PR TITLE
Removed Demographics Switch

### PR DESCRIPTION
**Pull request by clintonb**
The data is now available. There is no longer a need for this switch. The acceptance tests were also updated.

@dsjen @rocha

### CODE REVIEW

File: acceptance_tests/mixins.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def setUp(self):
 
         api_url = API_SERVER_URL
         auth_token = API_AUTH_TOKEN
-        self.api_client = Client(api_url, auth_token=auth_token, timeout=5)
+        self.api_client = Client(api_url, auth_token=auth_token, timeout=10)
```
Comment (dsjen) [inline acceptance_tests/mixins.py:23]: Was there a timeout issue?

Comment (clintonb) [inline acceptance_tests/mixins.py:23]: Only locally.

File: acceptance_tests/test_course_enrollment_demographics.py [hunk 2]
```diff
@@ -25,22 +23,27 @@ def setUp(self):
...
-        total_enrollment = sum([datum['count'] for datum in self.demographic_data])
+        demographic_data = self.demographic_data_without_none
+
+        total_enrollment = sum([datum['count'] for datum in demographic_data])
         half_enrollments = total_enrollment * 0.5
         count_enrollments = 0
 
-        sorted_by_year = sorted(self.course.enrollment(self.demographic_type),
-                                key=lambda item: item['birth_year'], reverse=False)
+        data = sorted(demographic_data, key=lambda item: item['birth_year'], reverse=False)
```
Comment (dsjen) [inline acceptance_tests/test_course_enrollment_demographics.py:44]: `sorted_by_year` seemed to be clearer to me.

**Comment by dsjen** [comment]
Looks good to me!

**Pull request merged**